src/components/PurchaseOrder/POInvoices/POInvoicesContainer.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Accordion } from '@folio/stripes/components';
+
+import {
+  INVOICES,
+  ORDER_INVOICES,
+} from '../../Utils/resources';
+
+import POInvoices from './POInvoices';
+
+class POInvoicesContainer extends Component {
+  static manifest = Object.freeze({
+    orderInvoicesRelns: ORDER_INVOICES,
+    invoices: INVOICES,
+  });
+
+  static propTypes = {
+    orderId: PropTypes.string.isRequired,
+    label: PropTypes.object.isRequired,
+    accordionId: PropTypes.string.isRequired,
+    mutator: PropTypes.shape({
+      orderInvoicesRelns: PropTypes.object.isRequired,
+      invoices: PropTypes.object.isRequired,
+    }).isRequired,
+    vendors: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  static defaultProps = {
+    vendors: [],
+  };
+
+  state = {
+    orderInvoices: [],
+  }
+
+  componentDidMount() {
+    this.fetchOrderInvoices();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.orderId !== prevProps.orderId) {
+      this.fetchOrderInvoices();
+    }
+  }
+
+  fetchOrderInvoices = () => {
+    const { mutator, orderId } = this.props;
+
+    mutator.orderInvoicesRelns.reset();
+    mutator.orderInvoicesRelns.GET({
+      params: {
+        query: `purchaseOrderId==${orderId}`,
+      },
+    }).then(list => list.map(item => item.invoiceId))
+      .then(invoicesIds => {
+        mutator.invoices.reset();
+
+        return invoicesIds.length
+          ? mutator.invoices.GET({
+            params: {
+              query: invoicesIds.map(id => `id==${id}`).join(' or '),
+            },
+          })
+          : [];
+      })
+      .then(orderInvoices => this.setState({ orderInvoices }))
+      .catch(() => {});
```

Comment:
such `catch` makes no sense, IMHO. I think it's better to handle that somehow or to not write at all

Reply:
Removed `catch` at all for now